examples/quickstart/notebooks/quickstart.ipynb

```diff
@@ -863,44 +903,41 @@
+    "a test set, registered the best one with the MLflow model registry, deployed it\n",
+    "and served some predictions. You also learned how to iterate on your models and\n",
+    "data by using some of the ZenML utility abstractions. You saw how to view your\n",
+    "artifacts and stacks via the CLI as well as the ZenML Dashboard.\n",
+    "\n",
+    "And that is just the tip of the iceberg of what ZenML can do; check out the [**Integrations**](https://zenml.io/integrations) page for a list of all the cool MLOps tools that ZenML supports!\n",
     "\n",
     "To improve upon the ML workflows we built in this quickstart, you could, for instance:\n",
-    "- [Deploy ZenML on the Cloud]() to collaborate with your teammates,\n",
+    "- [Deploy ZenML on the Cloud](https://docs.zenml.io/platform-guide/set-up-your-mlops-platform/deploy-zenml) to collaborate with your teammates,\n",
     "- Experiment with more sophisticated models, such as [XGBoost](https://zenml.io/integrations/xgboost),\n",
+    "- Monitor your model's ongoing performance using [Evidently](https://zenml.io/integrations/evidently),\n",
     "- Set up automated [Slack alerts](https://zenml.io/integrations/zen-ml-slack-integration) to get notified when data drift happens,\n",
-    "- Run the pipelines on scalable, distributed stacks like [Kubeflow](https://zenml.io/integrations/kubeflow)."
+    "- Run the pipelines on scalable, distributed stacks like\n",
+    "  [Kubeflow](https://zenml.io/integrations/kubeflow)."
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Where to go next\n",
+    "## What to do now\n",
     "\n",
     "* If you have questions or feedback... \n",
     "  * Join our [**Slack Community**](https://zenml.io/slack-invite) and become part of the ZenML family!\n",
-    "* If this quickstart was a bit too quick for you... \n",
-    "  * Check out [**ZenBytes**](https://github.com/zenml-io/zenbytes), our lesson series on practical MLOps, where we cover each MLOps concept in much more detail.\n",
-    "* If you want to learn more about using or extending ZenML...\n",
-    "  * Check out our [**Docs**](https://docs.zenml.io/) or read through our code on [**Github**](https://github.com/zenml-io/zenml).\n",
-    "* If you want to quickly learn how to use a specific tool with ZenML...\n",
-    "  * Check out our collection of [**Examples**](https://github.com/zenml-io/zenml/tree/doc/hamza-misc-updates/examples).\n",
-    "* If you want to see some advanced ZenML use cases... \n",
-    "  * Check out [**ZenML Projects**](https://github.com/zenml-io/zenml-projects), our collection of production-grade ML use-cases."
+    "* If you want to try ZenML in a real-world setting...\n",
+    "  * Check out the [ZenML Sandbox](https://sandbox.zenml.io/), a free trial of ZenML that runs on our Cloud\n",
+    "    platform. [**Sign up here**](https://sandbox.zenml.io/)."
```

Comment:
Maybe the main CTA should now be to sign up to the cloud instead? What's the status of sandbox actually, isn't this deprecated in favor of the cloud now?

Reply:
Written before the cloud was a thing :) I've updated now so that everything points to the cloud.